dashboard/test/models/pd/async_workshop_handler_test.rb

```diff
@@ -0,0 +1,55 @@
+require 'test_helper'
+require 'fake_sqs/test_integration'
+
+# Launch a fake SQS service running on Localhost unless an environment variable is set to use the
+# actual SQS service.
+unless ENV['USE_REAL_SQS']
```

Comment:
I copied this from [ActivityTest](https://github.com/code-dot-org/code-dot-org/blob/26f099b7c4797c28f317bef05fd624dcf8c31198/dashboard/test/models/activity_test.rb#L10), however I haven't actually used that env variable and don't foresee using it. When I tested with a real SQS queue, I just set it up in locals.yml and ran some manual tests. I'm going to remove this option and simplify the code.